### ISSUE TITLE
Incorrect file path for several paths in the project

### ISSUE THREAD

**Issue by antage** [AtomLinter/linter-elixirc#23]
When I add several project folder, linter-elixirc show error message with strange path:
```
Error: ENOENT: no such file or directory, scandir '[frist project path is here]/_build/dev/lib'
```
`_build/dev/lib` is path from second project folder.

I think the problem is here:
``` coffee
projectPath = ->
      atom.project.getPaths()[0]
```
We shouldn't use 0 index but we should detect active project folder and use it.

**Comment by antage** [comment]
The error backtrace:
```
Error: ENOENT: no such file or directory, scandir '/home/antage/workspace/chef/_build/dev/lib'
    at Error (native)
    at Object.fs.readdirSync (fs.js:813:18)
    at Object.fs.readdirSync (ATOM_SHELL_ASAR.js:449:28)
    at getDepsPa (/home/antage/.atom/packages/linter-elixirc/lib/init.coffee:98:10)
    at /home/antage/.atom/packages/linter-elixirc/lib/init.coffee:104:7
    at Object.provider.lint (/home/antage/.atom/packages/linter-elixirc/lib/init.coffee:119:11)
    at promises.push.Promise.then._this.emitter.emit.linter (/home/antage/.atom/packages/linter/lib/linter-registry.js:63:26)
    at /home/antage/.atom/packages/linter/lib/linter-registry.js:62:23
    at Set.forEach (native)
    at LinterRegistry.lint (/home/antage/.atom/packages/linter/lib/linter-registry.js:60:18)
    at /home/antage/.atom/packages/linter/lib/linter.coffee:98:16
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
    at /home/antage/.atom/packages/linter/lib/editor-linter.js:30:20
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:86:11)
    at TextBuffer.module.exports.TextBuffer.saveAs (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1035:20)
    at TextBuffer.module.exports.TextBuffer.save (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1002:19)
    at TextEditor.module.exports.TextEditor.save (/usr/share/atom/resources/app.asar/src/text-editor.js:557:26)
    at Pane.module.exports.Pane.saveItem (/usr/share/atom/resources/app.asar/src/pane.js:530:18)
    at Pane.module.exports.Pane.saveActiveItem (/usr/share/atom/resources/app.asar/src/pane.js:513:19)
    at Workspace.module.exports.Workspace.saveActivePaneItem (/usr/share/atom/resources/app.asar/src/workspace.js:631:35)
    at atom-workspace.commandRegistry.add.core:save (/usr/share/atom/resources/app.asar/src/register-default-commands.js:173:32)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:260:29)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:533:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:356:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:99:36)
    at HTMLDocument.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:3:61)
```
